integration-test/src/test/scala/com/cloudera/livy/test/BatchIT.scala

```diff
@@ -64,7 +80,7 @@ class BatchIT extends BaseIntegrationTestSuite {
 
   pytest("submit a pyspark application") {
     val hdfsPath = uploadResource("pytest.py")
-    val output = newOutputPath()
+    val output = newOutputPath
```

Comment:
No need to change these things, and I really prefer method calls to have the `()` in them.